### PR TITLE
Release v5.2 changes

### PR DESCRIPTION
**Pull request by Subasri-Viswanathan**
Included Upgrade changes

### CODE REVIEW

File: upgrade/upgrade.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Upgrading Bold Reports to latest version
+
+This section explains how to upgrade Bold Reports to latest version in your Kubernetes cluster. You can refer to the features and enhancements from this [Release Notes](https://www.boldreports.com/release-history/embedded-reporting).
+
+* [Upgrade Bold Reports to latest version](/upgrade/5-2_upgrade.md)
```
Comment (Sankaralingam1667) [inline upgrade/upgrade.md:5]: Upgrade Bold Reports from v5.1 to latest

File: upgrade/upgrade.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+* [Upgrade Bold Reports to latest version if your current version is below 5.1](/upgrade/5-1_upgrade.md)
```
Comment (Sankaralingam1667) [inline upgrade/upgrade.md:6]: Upgrade Bold Reports from older version (below v5.1) to latest

**Review by Sankaralingam1667 — CHANGES_REQUESTED** [review]

File: upgrade/upgrade.md (hunk 1 above)
Comment (Subasri-Viswanathan) [reply upgrade/upgrade.md]: changed

File: upgrade/upgrade.md (hunk 2 above)
Comment (Subasri-Viswanathan) [reply upgrade/upgrade.md]: changed

**Review by Sankaralingam1667 — APPROVED** [review]

**Pull request merged**